packages/client-direct/src/index.ts

```diff
@@ -69,6 +71,9 @@ export class DirectClient {
         const apiRouter = createApiRouter(this.agents, this);
         this.app.use(apiRouter);
 
+        const apiLogRouter = createVerifiableLogApiRouter(this.agents, this);
```

Comment:
If `VERIFIABLE_LOGGING` is not enabled, will this fail to start the agent? or will it just fail the API call?

Reply:
fail  the api call